scripts/automation/s3-test/create_account.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+# ./create_s3_account.sh username email ldap_user ldap_pwd
+
+S3_USER=$1
+BUILD_NUMBER=$2
+CORTX_USER_NAME=$3
+CORTX_PASSWORD=$4
+
+add_common_separator() {
+    echo -e '\n--------------- '"$*"' ---------------\n'
+}
+
+check_status() {
+    return_code=$?
+    error_message=$1
+    if [ $return_code -ne 0 ]; then
+        add_common_separator ERROR: $error_message
+    exit 1
+    fi
+    add_common_separator SUCCESS
+}
+
+if [ "$4x" == "x" ]; then
+    echo "Please pass proper input"
```

Comment:
use `provide` instead of `pass` and instead of input, if possible mention that which specific input parameter needed to be provided correctly? in this case I guess password. so you can mention that input param.